packages/glsp-playwright-core/src/integration/wait.fixes.ts

```diff
@@ -0,0 +1,9 @@
+// TODO: Workaround because of VSCode - VSCode does not really work well with page.waitForFunction
```

Comment:
Is there still something to be done or is it just a restriction that we have?

Reply:
A restriction, maybe we can find a solution for that in the future.. The problem arises if `iframes` are used.

Reply:
Noted, thanks for addition more comments!